app/tests/accessibility.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { configureAxe, getViolations, injectAxe } from "axe-playwright";
+import { expect, test } from "@playwright/test";
+import analyses from "@/fixtures/analyses.json" with { type: "json" };
+import { log } from "./util";
+
+log();
+
+/** pages to test */
+const paths = [
```

Comment:
Accessibility checks. Simply (for now) goes through a list of routes, and checks the whole page for accessibility issues. Essentially like if you were to manually run lighthouse in chrome dev tools on each page.